store/conf/attrs/zimbra-attrs.xml

```diff
@@ -8946,19 +8946,19 @@ TODO: delete them permanently from here
   <desc>Subject prefix for the spam training messages used to sent to the zimbraSpamIsSpamAccount/zimbraSpamIsNotSpamAccount account.</desc>
 </attr>
 
-<attr id="2021" name="zimbraDataSourceOAuthClientId" type="string" cardinality="single" optionalIn="dataSource,imapDataSource" since="8.7.0,9.0.0">
+<attr id="2021" name="zimbraDataSourceOAuthClientId" type="string" cardinality="single" optionalIn="globalConfig,domain,dataSource,imapDataSource" since="8.7.0,9.0.0">
```

Comment:
Probably need upgrade testing to make sure that anything working before an upgrade continues to work.

Reply:
@fatkudu , I'll retract this PR, and discuss about the design in project. Thank you for your input.